modules/serverless-send-events-kn.adoc

```diff
@@ -1,9 +1,11 @@
 :_content-type: PROCEDURE
 [id="serverless-send-events-kn_{context}"]
-= Sending events
+= Sending events by using the kn event plug-in
 
 You can use the `kn event send` command to send an event. The events can be sent either to publicly available addresses or to addressable resources inside a cluster, such as Kubernetes services, as well as Knative services, brokers, and channels. The command uses the same builder-like interface as the `kn event build` command.
 
+// prereqs?
```

Comment:
The kn-event is bundled with Serverless distribution of `kn` CLI. This means if the user has Serverless `kn` installed, there's nothing else he needs to do.

Maybe it is a good idea to mention, we are delivering `kn event` with our version of `kn`? Just to clarify.